include/mbgl/storage/default_file_source.hpp

```diff
@@ -115,7 +115,23 @@ class DefaultFileSource : public FileSource {
      */
     void setOfflineMapboxTileCountLimit(uint64_t) const;
 
+    /*
+     * Pause file request activity.
+     *
+     * If pause is called then no revalidation or network request activity
+     * will occur.
+     *
+     * Note: Calling pause and then calling getAPIBaseURL or getAccessToken
```

Comment:
fyi this was fixed in https://github.com/mapbox/mapbox-gl-native/pull/8465 @tmpsantos